From 96465a6e47829b2a9867dd2beddd7ae3a16b47bd Mon Sep 17 00:00:00 2001
From: Jan Kiszka <jan.kiszka@siemens.com>
Date: Mon, 13 Sep 2021 08:29:28 +0200
Subject: [PATCH 7/8] python3: Drop bogus CMAKE_C_FLAGS
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

CMake automatically pulls CMAKE_C_FLAGS and CMAKE_CXX_FLAGS, according
to the source file. Adding CMAKE_C_FLAGS unconditionally only raises

cc1plus: warning: ‘-Werror=’ argument ‘-Werror=implicit’ is not valid for C++
cc1plus: warning: ‘-Werror=’ argument ‘-Werror=missing-parameter-type’ is not valid for C++

Signed-off-by: Jan Kiszka <jan.kiszka@siemens.com>
---
 src/python/python3/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/python/python3/CMakeLists.txt b/src/python/python3/CMakeLists.txt
index 03894cf..c8de846 100644
--- a/src/python/python3/CMakeLists.txt
+++ b/src/python/python3/CMakeLists.txt
@@ -25,7 +25,7 @@ endif ()
 
 set_target_properties (${SWIG_MODULE_python3-mraa_REAL_NAME} PROPERTIES
   OUTPUT_NAME _mraa
-  COMPILE_FLAGS "${CMAKE_C_FLAGS} -DSWIGPYTHON=${SWIG_FOUND} -DSWIGPYTHON3=${SWIG_FOUND}"
+  COMPILE_FLAGS "-DSWIGPYTHON=${SWIG_FOUND} -DSWIGPYTHON3=${SWIG_FOUND}"
 )
 
 install (TARGETS ${SWIG_MODULE_python3-mraa_REAL_NAME}
-- 
2.32.0

